File: Iniciante/1074.py
# -*- coding: utf-8 -*-

N = int (input())
x = 0
for i in  range (1, N+1):
    x = int (input())
    if ( x > 0 ):
        if ( x % 2 == 0):
            print('EVEN POSITIVE')
        else:
            print ('ODD POSITIVE')
    elif ( x == 0):
        print ('NULL')
    elif ( x < 0 ):
        if ( x % 2 == 0):
            print('EVEN NEGATIVE')
        else:
            print ('ODD NEGATIVE')

File: Iniciante/1074.c
#include <stdio.h>

int main(void) {
	int N = 0, number = 0;

	scanf("%d", &N);
	for (int i = 0; i < N; i++) {
		scanf("%d", &number);
		if (number % 2 == 0 && number > 0) {
			printf("EVEN POSITIVE\n");
		} else if (number % 2 == 0 && number < 0) {
			printf("EVEN NEGATIVE\n");
		} else if (number % 2 != 0 && number > 0) {
			printf("ODD POSITIVE\n");
		} else if (number % 2 != 0 && number < 0) {
			printf("ODD NEGATIVE\n");
		} else {
			printf("NULL\n");
		}
	}
	return 0;
}
